ui/partials/perf/testdatachooser.html

```diff
@@ -44,5 +44,5 @@ <h3 class="modal-title">Add test data</h3>
   </div>
 </div>
 <div class="modal-footer">
-  <button class="btn btn-primary" ng-click="addTestData()" ng-disabled="!addedTestList.length" ng-model="rightList"><span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Add</button>
+  <button class="btn btn-primary" ng-click="addTestData()" ng-disabled="!addedTestList.length" ng-model="rightList"><span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Add ({{addedTestList.length}})</button>
```

Comment:
This isn't necessary, we can leave it out. It's already obvious how many tests are going to be added from glancing at the list. :)